images/capi/packer/azure/packer.json

```
@@ -254,6 +254,7 @@
     "kubernetes_semver": null,
     "kubernetes_series": null,
     "kubernetes_source_type": null,
+    "managed_image_storage_account_type": "",
```

To ensure this doesn't break for anyone it might be worth defaulting this to the `storage_account_type` value.
```suggestion
    "managed_image_storage_account_type": "{{user `storage_account_type`}}",
```

---

make sense , thanks :+1: